src/main/java/com/depromeet/buzz/post/domain/Post.java

```diff
@@ -43,36 +50,54 @@
 	@JoinColumn(name = "user_id")
 	private User user;
 
-	private Post() { }
+	@OneToMany(mappedBy = "post")
+	private List<Discount> discounts;
+
+	@OneToMany(mappedBy = "post")
+	private List<Participation> participations;
+
+	private Post() {}
 
-	public Post(String productName, int price, String thumbnail,
-		String detailPage, Date regDate, Date closingDate) {
+	public Post(String productName, int price, String thumbnail, String detailPage
+		, Date regDate, Date closingDate, Category category, User user,
+		List<Discount> discounts, List<Participation> participations) {
 		this.productName = productName;
 		this.price = price;
 		this.thumbnail = thumbnail;
 		this.detailPage = detailPage;
 		this.regDate = regDate;
 		this.closingDate = closingDate;
+		this.category = category;
+		this.user = user;
+		this.discounts = discounts;
+		this.participations = participations;
 	}
 
-	@Override
-	public boolean equals(Object o) {
-		if (this == o) {
-			return true;
-		}
+	public int findDiscountRateByUserCount() {
+		int userCount = participations.size();
 
-		if (o == null || getClass() != o.getClass()) {
-			return false;
-		}
+		Discount discount = findDiscountByUserCount(userCount);
+		return discount == NOT_DISCOUNT ? Step.ZERO.getStep() : discount.getDiscountRate();
+	}
+
+	public Discount findDiscountByUserCount(int userCount) {
+		return discounts.stream()
+			.filter(x -> x.getStepByUserCount(userCount) > Step.ZERO.getStep())
```

Comment:
```suggestion
			.filter(x -> x.getStepByUserCount(userCount) > Step.ZERO.getStep())
```

x.isNotZeroStep()과 같이 비교하는 검사를 Discount쪽에 만드는 건 어떨까요??? get을 최대한 지양!